### ISSUE TITLE
Fix false positives for shared-line comments separated by more than once space in selector-list-comma-newline-after

### ISSUE THREAD

**Issue by pkuczynski** [stylelint/stylelint#2912]
With default settings rule `selector-list-comma-newline-after` complaints about following CSS:

```
.myClass,  // this is comment
.myOtherClass {
  ...
}
```

Error: `Expected newline after "," `

But does not complain when there is a single space before the comment. I might attempt to create a PR for this, if you agree with me, that it needs fixing...

**Comment by jeddy3** [comment]
@pkuczynski Thanks for the report.

>  if you agree with me, that it needs fixing...

Yes, I think so, as the beauty about comments is that they can be put anywhere. At the moment [we're assuming](https://github.com/stylelint/stylelint/blob/master/lib/rules/selector-list-comma-newline-after/index.js#L52) there [will only be one space](https://github.com/stylelint/stylelint/blob/master/lib/rules/selector-list-comma-newline-after/index.js#L59) before a shared-line comment, and I don't see the harm in making that more lenient to any number of spaces.

> I might attempt to create a PR for this

That'd be great! There's a guide [here](https://github.com/stylelint/stylelint/blob/master/docs/developer-guide/rules.md#fixing-a-bug-in-an-existing-rule).

---

In the future, please use the issue template as it helps the team scan through issues quicker.

**Comment by pkuczynski** [comment]
Here it comes: #2915

As this is my first PR in this project, I am open to hearing your feedback. If all is good, I would fix #2906.
